modules/custom/az_paragraphs/az_paragraphs_text_media/js/az_paragraphs_az_text_media_vimeo.es6.js

```diff
@@ -0,0 +1,189 @@
+    attach(context, settings) {
+      if (window.screen && window.screen.width > 768) {
+        // @see https://developer.vimeo.com/player/sdk/basics
+        // Defaults
+        const defaults = {
+          ratio: 16 / 9,
+          vimeoId: '',
+          mute: true,
+          repeat: true,
+          width: $(window).width(),
+          playButtonClass: 'az-video-play',
+          pauseButtonClass: 'az-video-pause',
+          start: 0,
+          minimumSupportedWidth: 600,
+        };
+        const { bgVideos } = settings.azFieldsMedia;
+        const bgVideoParagraphs = document.getElementsByClassName(
+          'az-js-vimeo-video-background',
+        );
+        // Load Vimeo API
+        const tag = document.createElement('script');
+        tag.src = 'https://player.vimeo.com/api/player.js';
+        document.head.appendChild(tag);
+
+        // Methods
+        // Ensure Vimeo API is loaded before proceeding
+        tag.onload = () => {
+          $.each(bgVideoParagraphs, (index) => {
+            const thisContainer = bgVideoParagraphs[index];
+            const parentParagraph = thisContainer.parentNode;
+            const vimeoId = thisContainer.dataset.vimeoId2;
+            bgVideos[vimeoId] = $.extend({}, defaults, thisContainer);
+            const options = bgVideos[vimeoId];
+            const videoPlayer =
+              thisContainer.getElementsByClassName('az-video-player')[0];
+            const VimeoPlayer = window.Vimeo;
+
+            // Initialize Vimeo Player
+            thisContainer.player = new VimeoPlayer.Player(videoPlayer, {
+              id: vimeoId,
+              width: options.width,
+              height: Math.ceil(options.width / options.ratio),
+              autoplay: true,
+              muted: options.mute,
+              loop: options.repeat,
+              background: options.background,
+              title: options.title,
+              byline: options.byline,
+              portrait: options.portrait,
+            });
+
+            // Event Listeners
+            thisContainer.player.on('play', () => {
+              parentParagraph.classList.add('az-video-playing');
+              parentParagraph.classList.remove('az-video-paused');
+            });
+
+            thisContainer.player.on('pause', () => {
+              parentParagraph.classList.add('az-video-paused');
+              parentParagraph.classList.remove('az-video-playing');
+            });
+
+            thisContainer.player.on('ended', () => {
+              if (options.repeat) {
+                thisContainer.player.setCurrentTime(0).then(() => {
+                  thisContainer.player.play();
+                });
+              }
+            });
+
+            // Play Button
+            const playButton =
+              bgVideoParagraphs[index].getElementsByClassName(
+                'az-video-play',
+              )[0];
+            playButton.addEventListener('click', async (event) => {
```

Comment:
I think we can remove the async declaration here and with the event listener for pauseButton? Playing and pausing seemed to work fine without it and all the minified JS is no longer added to the .js file when you run `lando yarn build`.

```suggestion
            playButton.addEventListener('click', (event) => {
```